### PR TITLE
Functional tests - Refacto 'BO > International > Translations' tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto 'BO > International > Translations' tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI and nightly are :green_heart: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/6814673239
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/catalog/discounts/add.ts [hunk 1]
```diff
@@ -477,6 +489,17 @@ class AddCartRule extends BOBasePage {
     }
   }
 
+  /**
+   * Get title of exclude discounted product
+   * @param page
+   */
+  async getTitleOfExcludeDiscountedProduct(page: Page): Promise<string> {
+    // Go to actions tab
+    await page.click(this.actionsTabLink);
```
Comment (Progi1984) [inline tests/UI/pages/BO/catalog/discounts/add.ts]:
```suggestion
    await page.locator(this.actionsTabLink).click();
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/pages/BO/catalog/discounts/add.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/catalog/discounts/add.ts]: Done  thx:+1:

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by nicosomb** [comment]
Thank you @nesrineabdmouleh and @Progi1984 !

**Pull request merged**